<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PVZCheaterClass</class>
 <widget class="QMainWindow" name="PVZCheaterClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PVZCheater</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lable_sunshine">
        <property name="text">
         <string>阳光：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_nowSunshine">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_editSunshine"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_setSunshine">
        <property name="text">
         <string>修改</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>功能</string>
    </property>
    <addaction name="action_refresh"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action_refresh">
   <property name="text">
    <string>刷新</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PVZCheater.qrc"/>
 </resources>
 <connections/>
</ui>
